src/PrestaShopBundle/Service/Log/LogHandler.php

```diff
@@ -50,13 +44,15 @@ public function __construct(Container $container, $level = Logger::DEBUG, $bubbl
 
     /**
      * Writes the record down to the log of the implementing handler
-     *
-     * @phpstan-param FormattedRecord $record
      */
-    protected function write(array $record): void
+    protected function write(LogRecord $record): void
```

Comment:
Was the update of monolog that forces this modification a requirement?

Reply:
Yes, see https://github.com/Seldaek/monolog/blob/main/UPGRADE.md